lumapps/api/client.py

```diff
@@ -376,29 +389,42 @@ def get_call(self, *name_parts, **params):
         """
         name_parts = _parse_endpoint_parts(name_parts)
         items = []
-        cursor = None
+        self.cursor = cursor
```

Comment:
Why must this become an object-level variable?

Reply:
To be able to get it after a call failed for instance.